Dockerfile

```diff
@@ -4,8 +4,6 @@ RUN mkdir -p /app
 
 WORKDIR /app
 
-RUN npm install npm@^8 --location=global
```

Comment:
Doesn't solve the problem but I think we can get rid of this anyway. I added it before because I saw a warning about the `package-lock.json` made in an earlier version and to do the one-time upgrade, etc.